src/gsUtils/gsQuasiInterpolate.hpp

```diff
@@ -93,6 +93,13 @@ gsMatrix<T> gsQuasiInterpolate<T>::localIntpl(const gsBasis<T> &bb,
     if (const gsHTensorBasis<3,T>* b = dynamic_cast<const gsHTensorBasis<3,T>* >(&bb))
         return localIntpl(*b,fun,i);
     if (const gsHTensorBasis<4,T>* b = dynamic_cast<const gsHTensorBasis<4,T>* >(&bb))
+        return localIntpl(*b,fun,i);        
+    // If it is a gsRationalTHBSplineBasis, we check the source
```

Comment:
Where do we check specifically for gsRationalTHBSplineBasis? I only see gsHTensorBasis

Reply:
Below the comment line [97] we check if the `.source()` is an `gsHTensorBasis`, which is the case for rational THB splines